packages/registry-contracts/script/test/deploy-root-verifier.sh

```
@@ -0,0 +1,54 @@
+  exit 1
+fi
+
+# Deploy all verifier contracts (RootVerifier + VerifierHelper + SubVerifier + proof verifiers)
+echo "Deploying ZKPassport verifier stack..."
+forge script script/DeployRootVerifier.s.sol:DeployRootVerifierScript \
+  --rpc-url $RPC_URL \
+  --private-key $DEPLOYER_PRIVATE_KEY \
+  --broadcast
+
+# Extract the RootVerifier address (first CREATE transaction)
+export ROOT_VERIFIER_ADDRESS=$(cat broadcast/DeployRootVerifier.s.sol/$CHAIN_ID/run-latest.json | jq -r '[.transactions[] | select(.transactionType=="CREATE")][0].contractAddress')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)